lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js

```diff
@@ -96,11 +97,17 @@ class NoVulnerableLibrariesAudit extends Audit {
       return [];
     }
 
-    const snykInfo = snykDB.npm[lib.npmPkgName];
-    const vulns = snykInfo
-      .filter(vuln => semver.satisfies(normalizedVersion, vuln.semver.vulnerable[0]))
-      // valid vulnerability
-      .map(vuln => {
+    // Match the vulnerability candidates from snyk against the version we see in the page
+    const vulnCandidatesForLib = snykDB.npm[lib.npmPkgName];
+    const matchingVulns = vulnCandidatesForLib.filter(vulnCandidate => {
+      // Each snyk vulnerability comes with an array of semver ranges.
```

Comment:
maybe `array of semver ranges, the page is vulnerable if any of the ranges match` or something? otherwise feels like it's just describing the type